packages/stanza-react/src/hooks/useStanzaContext.ts

```diff
@@ -0,0 +1,35 @@
+import { getContextStale, type StanzaContext } from '@getstanza/browser'
+import { useContext, useEffect, useState } from 'react'
+import { StanzaReactContext } from '../context/StanzaContext'
+import { StanzaContextName } from '../context/StanzaContextName'
+
+export const useStanzaContext = (contextName?: string): StanzaContext | undefined => {
+  const providedContextName = useContext(StanzaContextName)
+  const [state, setState] = useState<StanzaContext | undefined>(undefined)
```

Comment:
We don't have to explicitly pass undefined in this case:

```suggestion
  const [state, setState] = useState<StanzaContext | undefined>()
```

Reply:
done